lightningd/log.c

```diff
@@ -444,14 +446,67 @@ static void show_log_prefix(char buf[OPT_SHOW_LEN], const struct log *log)
 	strncpy(buf, log->prefix, OPT_SHOW_LEN);
 }
 
+static int signalfds[2];
+
+static void handle_sighup(int sig)
+{
+	/* This may fail if we're hammered with SIGHUP.  We don't care. */
+	if (write(signalfds[1], "", 1));
```

Comment:
Ok, this took me a while to figure out. This'll just write a 0x00 byte, since any C string is null-terminated, right?